ddht/v5_1/alexandria/client.py

```diff
@@ -253,6 +275,38 @@ async def send_pong(
         message = PongMessage(PongPayload(enr_seq))
         await self._send_response(node_id, endpoint, message, request_id=request_id)
 
+    async def send_find_nodes(
+        self,
+        node_id: NodeID,
+        endpoint: Endpoint,
+        *,
+        distances: Collection[int],
+        request_id: Optional[bytes] = None,
+    ) -> bytes:
+        message = FindNodesMessage(FindNodesPayload(tuple(distances)))
```

Comment:
I know I already approved this approach back in #116 but seeing this line is starting to give me some pause. Why are the payloads separate classes than the messages? Lines like this and methods like `from_payload_args` seem like they're workarounds to the decision to separate payloads from messages.

Reply:
Keeping the message and payload separate was intended to separate metadata from payload.

- metadata
    - message type flag
    - SSZ sedes to know how to encode/decode the payload
- payload
    - the actual data for the message

Admittedly, the metadata can be class-level properties since they don't change from message to message and the payloads are the dynamic parts that only exist at the instance level...

This design choice was more of a whim which was inspired by thinking certain things would be cleaner with the two separate.  I'm not sure it's better..  I'm ok unwinding it at some point if there's no clear benefit.  I'll open an issue to track this.